src/Microsoft.ML.StandardLearners/Standard/MultiClass/doc.xml

```diff
@@ -14,10 +14,6 @@
         This multi-class trainer accepts binary feature values of type float, i.e., feature values are either true or false.
         Specifically a feature value greater than zero is treated as true.
       </remarks>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.LogisticRegressionClassifier'/>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.LightGbmClassifier'/>
-      <seealso cref='Microsoft.ML.Legacy.Trainers.StochasticDualCoordinateAscentClassifier'/>
-      <seealso cref='Microsoft.ML.Legacy.Models.OneVersusAll'/>
```

Comment:
Could we keep a reference to the trainers here? The trainers with the new API

Reply:
@sfilipi had asked to swap these names out in the previous PR.